.github/actions/pr-status-updater/action.yml

```
@@ -0,0 +1,68 @@
+# Copied / Extracted from `code-freeze/action.yml`:
+
+name: 'PR Status Updater'
+description: 'Updates PR statuses for code freeze'
+
+inputs:
+  github_token:
+    description: 'GitHub token with PR and status management permissions'
+    required: true
+  state:
+    description: 'Status state to set (success/failure)'
+    required: true
+  workflow_name:
+    description: 'Name of the workflow file for the target URL'
+    required: true
+  page_size:
+    description: 'Number of PRs to process per page'
+    required: false
+    default: '25'
+  max_pages:
+    description: 'Maximum number of pages to process'
+    required: false
+    default: '5'
```

We should probably bump this, seeing as we already have 127 PRs 😅 
```suggestion
    default: '6'
```